home/home_10.1/tournament.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+class Tournament {
+    var teams: [Team] = []
+    var matches: [Match] = []
+
+    func addTeam(_ team: Team) {
+        teams.append(team)
+    }
+
+    func addMatch(_ match: Match) {
+        matches.append(match)
+    }
+
+    func calculateRanking() {
+        teams.sort { a, b in
+            if a.points != b.points { return a.points > b.points }
+            if a.topHeights != b.topHeights { return a.topHeights > b.topHeights }
+            if a.goalDelta() != b.goalDelta() { return a.goalDelta() > b.goalDelta() }
+            if a.goalsScored != b.goalsScored { return a.goalsScored > b.goalsScored }
+            if a.redCards != b.redCards { return a.redCards < b.redCards }
+            if a.yellowCards != b.yellowCards { return a.yellowCards < b.yellowCards }
+            return Bool.random()
+        }
+
+        if let russiaIndex = teams.firstIndex(where: { $0.name.lowercased() == "russia" }) {
+            let russiaTeam = teams.remove(at: russiaIndex)
```

Comment:
minor: it can be a part of sort logic as a first check - if a.russia => false, if b.russia => true